components/search/themes/spectrum.css

```diff
@@ -0,0 +1,56 @@
+/*
+Copyright 2023 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+@container (--system: spectrum) {
+    .spectrum-Search {
+        --spectrum-search-border-radius: var(--spectrum-corner-radius-100);
+        --spectrum-search-edge-to-visual: var(--spectrum-component-edge-to-visual-100);
+        --spectrum-search-edge-to-text: var(--spectrum-component-edge-to-text-100);
+
+        --spectrum-search-border-color-default: var(--spectrum-gray-500);
+        --spectrum-search-border-color-hover: var(--spectrum-gray-600);
+        --spectrum-search-border-color-focus: var(--spectrum-gray-800);
+        --spectrum-search-border-color-focus-hover: var(--spectrum-gray-900);
+        --spectrum-search-border-color-key-focus: var(--spectrum-gray-900);
+    }
+
+    .spectrum-Search--sizeS {
+        --spectrum-search-border-radius: var(--spectrum-corner-radius-100);
+        --spectrum-search-edge-to-visual: var(--spectrum-component-edge-to-visual-75);
+        --spectrum-search-edge-to-text: var(--spectrum-component-edge-to-text-75);
+    }
+
+    .spectrum-Search--sizeM {
+        --spectrum-search-border-radius: var(--spectrum-corner-radius-100);
+        --spectrum-search-edge-to-visual: var(--spectrum-component-edge-to-visual-100);
+        --spectrum-search-edge-to-text: var(--spectrum-component-edge-to-text-100);
+    }
+
+    .spectrum-Search--sizeL {
+        --spectrum-search-border-radius: var(--spectrum-corner-radius-100);
+        --spectrum-search-edge-to-visual: var(--spectrum-component-edge-to-visual-200);
+        --spectrum-search-edge-to-text: var(--spectrum-component-edge-to-text-200);
+    }
+
+    .spectrum-Search--sizeXL {
+        --spectrum-search-border-radius: var(--spectrum-corner-radius-100);
+        --spectrum-search-edge-to-visual: var(--spectrum-component-edge-to-visual-300);
+        --spectrum-search-edge-to-text: var(--spectrum-component-edge-to-text-300);
+    }
+
+    .spectrum-Search--quiet {
+        --spectrum-search-border-radius: 0;
+        --spectrum-search-edge-to-text: var(--spectrum-field-edge-to-visual-quiet);
+        --spectrum-search-edge-to-visual: var(--spectrum-field-edge-to-visual-quiet);
+    }
```

Comment:
It looks like these custom properties are the same in Spectrum as they are in Express. Are we able to pull these out and place them into `index.css` rather than having them in these theme files?

Reply:
I've moved these back with the most recent commit. I originally had them in index.css, but they were being overridden by the system specific files (that get tacked on at the end). The adjustment needed was to increase specificity for those particular quiet properties in index.css.